fhir-server-test/src/test/java/org/linuxforhealth/fhir/client/test/FHIRClientTest.java

```diff
@@ -79,6 +84,67 @@ public void testFHIRClientCtorProperties2() throws Exception {
         assertNotNull(c);
         assertEquals(MIMETYPE_XML, c.getDefaultMimeType());
     }
+    
+    @Test
+    public void testFHIRClientMimeType() throws Exception {
+        // The default mimetype should specify FHIR version 4.3, so let's make
+        // sure that with the default settings we can create a 4.3-only resource
+        Ingredient ingredient = TestUtil.readLocalResource("Ingredient-1.json");
```

Comment:
we could use `TestUtil.getMinimalResource(Ingredient.class)` here instead and avoid the need for checking in this extra resource under src/test

Reply:
Fair point. But will skip this this time.